core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/ApiServiceSpec.scala

```diff
@@ -103,8 +103,8 @@ trait ApiServiceSpec extends TestDriverComponentWithFlatSpecAndMatchers with Raw
   }
 
   //noinspection TypeAnnotation,NameBooleanParameters,ConvertibleToMethodValue,UnitMethodIsParameterless
-  trait ApiServices extends AdminApiService with BillingApiService with EntityApiService with MethodConfigApiService
-    with NotificationsApiService with RawlsApiService with SnapshotApiService with StatusApiService with SubmissionApiService with UserApiService with WorkspaceApiService {
+  trait ApiServices extends AdminApiService with BillingApiService with EntityApiService
```

Comment:
`RawlsApiService` mixes in `MethodConfigApiService` and `SubmissionApiService` already